src/Database/Connection.php

```diff
@@ -471,20 +559,17 @@ public function rollback()
      * `$connection->useSavePoints(false)` Disables usage of savepoints and returns false
      * `$connection->useSavePoints()` Returns current status
      *
+     * @deprecated 3.4.0 Use enableSavePoints()/isEnabledSavePoints() instead.
      * @param bool|null $enable Whether or not save points should be used.
      * @return bool true if enabled, false otherwise
      */
     public function useSavePoints($enable = null)
     {
-        if ($enable === null) {
-            return $this->_useSavePoints;
-        }
-
-        if ($enable === false) {
-            return $this->_useSavePoints = false;
+        if ($enable !== null) {
+            $this->enableSavePoints($enable);
         }
 
-        return $this->_useSavePoints = $this->_driver->supportsSavePoints();
+        return $this->isEnabledSavePoints();
```

Comment:
Why create getters/setters and be nonconventional about them in the first place. But its a moot point, it will work either way.

But please if you move "Enabled" to the back of one of the getters, then do so for all of them, could that be possible?